src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -42,7 +46,8 @@ public EditCommand parse(String args) throws ParseException {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, EditCommand.MESSAGE_USAGE), pe);
         }
 
-        argMultimap.verifyNoDuplicatePrefixesFor(PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL, PREFIX_ADDRESS);
```

Comment:
Wait I thought there is no preference field anymore and they are combined with information?